Userland/Libraries/LibSoftGPU/Clipper.h

```diff
@@ -24,9 +24,15 @@ class Clipper final {
         FAR
     };
 
+    enum class Space : u8 {
+        CLIP = 0,
+        VIEW,
```

Comment:
I know the previous enum has all-caps items, but we generally don't uppercase our enum values.

Reply:
The latest commit does away with the `Space` enum so I have taken the liberty of also lowercasing the `ClipSpace` enum

Reply:
Great! But then we also usually PascalCase them, so `Left`, `Right` instead of `left`, `right`

Reply:
Changed to PascalCase in latest commit